### DISCUSSION TITLE
How did you make ElevenClock using Python?

### DISCUSSION THREAD

**Discussion by TechStudent10** [martinet101/ElevenClock#314] [Q&A] [answered]
As a person who has used Python ever since I started coding, I've never seen **ANYONE** use Python like this. It kind of seems impossible that you were able to make ElevenClock, **PURELY**, using Python with this many options.

So how did you make it? What libraries are you using? I am really just curious because before I checked the "Language (percentage of how much of the repo used that language", I thought you were using C# or something like that!

**Comment by marticliment** [comment, marked as answer]
Hi @TechStudent11, i'm a 16 y/o student and, not gonna lie, when i look at the code I don't really understand what I really did😅. The main trick of the app is placing a Qt frameless window over the taskbar and place it at the bottom corner of the screen. The key to get system settings such as formats, etc I just used Winreg, a module to read registry. And for the fullscreen hiding, i just used some bindings to the win32 api.

For the settings window, it is just a regular window with lots of stylesheet to make it look as Windows 11 UWP.

**Reply by TechStudent10** [reply]
@martinet101 Okay, but how did you get it on the second screen? Whenever I try to Google it, it doesn't seem possible.

**Reply by marticliment** [reply]
Let's say I have a 1920*1080 at 0x0 and another 1920*1080 at 1920*0 (Monitor next to other). I just have to add 1920+1920 and subtract the clock's width, which makes (1920+1920-200)=3840-200=3640px, and i move the clock to the 3640px on the x. For the Ys is the same thing

**Reply by TechStudent10** [reply]
@martinet101 Oh so it's just math. Okay, thanks!

**Reply by marticliment** [reply]
Yep, basically it is math
